src/templates/error.js

```diff
@@ -1,22 +1,20 @@
 export default (id, message, errorCode, code) => {
-    const detail = code ? `(${errorCode}: ${code})` : '';
+    message += code ? ` (${errorCode}: ${code})` : '';
     return (
         `<div id="${id}" class="jw-error jw-reset">` +
-            `<div class="jw-error-msg jw-reset">` +
+            `<div class="jw-error-msg jw-info-overlay jw-reset">` +
                 `<style>` +
-                `[id="${id}"].jw-error{position:relative;background:#000;overflow:hidden;position:relative}` +
-                `[id="${id}"] .jw-error-msg{top:50%;left:50%;position:absolute;align-items:center;display:flex;transform:translate(-50%,-50%)}` +
-                `[id="${id}"] .jw-title{color:#FFF;position:static}` +
-                `[id="${id}"] .jw-title-primary,` +
-                `[id="${id}"] .jw-title-secondary{font:600 14px/1.35 Arial,Helvetica,sans-serif}` +
-                `[id="${id}"] .jw-title-secondary{font-weight:400}` +
+                `[id="${id}"].jw-error{background:#000;overflow:hidden;position:relative}` +
+                `[id="${id}"] .jw-error-msg{top:50%;left:50%;position:absolute;align-items:center;transform:translate(-50%,-50%)}` +
+                `[id="${id}"] .jw-error-container{display:flex;flex-direction:column;align-items:center}` +
+                `[id="${id}"] .jw-error-text{color:#FFF;position:static;font:600 14px/1.35 Arial,Helvetica,sans-serif}` +
+                `[id="${id}"] .jw-error-text{font-weight:400}` +
```

Comment:
Combine `.jw-error-text` rules into the first line `font:400 14px/1.35 Arial` (or just delete the font weight value: `font:14px/1.35 Arial`)

Reply:
Will do